ha/core/system_health/system_health_metadata.py

```diff
@@ -105,6 +105,8 @@ class SystemHealthHierarchy:
     _server_service = [const.COMPONENTS.SERVER_SERVICE.value, const.COMPONENTS.AGG_SERVICE.value] + _server
     _server_hw = [const.COMPONENTS.SERVER_HARDWARE.value] + _server
     _storage = [const.COMPONENTS.STORAGE_COMPONENT.value,  const.COMPONENTS.STORAGE.value] + _common_hierarchy
+    _cvg = [const.COMPONENTS.CVG.value, const.COMPONENTS.RACK.value, const.COMPONENTS.SITE.value,
```

Comment:
Can't this hierarchy should be
cluster>site>rack>node>cvg?
When this will be executed, you will have key ike:
cluster:<cluster_id>:site:<site-id>:rack:<rack-id>:cvg:<cvg-id>. Right?
Don't we need node_id also? (for CVG assosciation with particular node) ?

Reply:
__cvg = [const.COMPONENTS.CVG.value, const.COMPONENTS.RACK.value, const.COMPONENTS.SITE.value, const.COMPONENTS.CLUSTER.value]_
This list is only for hierarchy update, means if there is change in cvg health  then which all components gets affected are listed here. (CVGs & Nodes are independent)

>>>When this will be executed, you will have key ike:
>>>cluster:cluster_id:site: site-id: rack:rack-id: cvg:cvg-id. Right?

No, Key will be created as defined in SystemHealthComponents class i.e. "/cortx/ha/system/cluster/$cluster_id/site/$site_id/rack/$rack_id/node/$node_id/cvg/$cvg_id/health"
e.g.  cortx/ha/v1/cortx/ha/system/cluster/7020f6ba756e466fa20962d848ddaa76/site/0xBAADFOOD/rack/0xBAADFOOD/node/42db803c061646f4871b42eb74b9b2b8/cvg/cvg-02/health

>>>Don't we need node_id also? (for CVG assosciation with particular node) ?

We need node_id to create keys.

Reply:
Ahh Ok..Got it..Hierarchy you are already picking up from SystemHealthComponents._componenets which has node_id in the cvg key. The change which I was expecting is for updating node in case of cvg health change. Which is not required now. Thanks!